rdkit/Chem/PandasTools.py

```diff
@@ -252,13 +301,24 @@ def PrintAsBase64PNGString(x, renderer=None):
         highlightAtoms = x.__sssAtoms
     else:
         highlightAtoms = []
+    # TODO: should we generate coordinates if no coordinates available?
+    # from rdkit.Chem import rdDepictor
+    # try:
+    #     # If no coordinates, calculate 2D
+    #     x.GetConformer(-1)
+    # except ValueError:
+    #     rdDepictor.Compute2DCoords(x)
     if molRepresentation.lower() == 'svg':
-        from IPython.display import SVG
         svg = Draw._moltoSVG(x, molSize, highlightAtoms, "", True)
-        return SVG(svg).data
+        svg = minidom.parseString(svg)
+        svg = svg.getElementsByTagName('svg')[0]
+        svg.attributes['viewbox'] = f'0 0 {molSize[0]} {molSize[1]}'
```

Comment:
Awesome.  I tend to think we should add the view box to the underlying drawing code.  If greg approves I'm happy to push a PR to your repo to add this.

Reply:
@bp-kelley yes, that makes sense to me.
Please add an issue for it as well so that we end up with something in the release notes.